_posts/2014-06-26-performance-improvment-hybrid-query-215.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title:  "Boosting Hybrid Query Performance in OpenSearch 2.15"
+authors:
+  - gaievski
+  - vamshin
+  - macrakis
+date: 2024-07-02
+categories:
+  - technical-posts
+meta_keywords: Multimodal search, searching with semantic and visual understanding, improve search relevance, hybrid search in OpenSearch 2.10, keyword and image search
+meta_description: Improve hybrid query performance in OpenSearch 2.15.
```

Comment:
Please update the meta keywords and meta description with the following: 

_meta_keywords:_ hybrid search, boost hybrid query performance, OpenSearch 2.15, semantic search 

_meta_description:_ The launch of OpenSearch 2.15 greatly improves hybrid search with numerous enhancements that boost hybrid query performance by as much as 70% in comparison to version 2.13.

Reply:
ack